src/pages/api/og/post/[postId].tsx

```diff
@@ -75,37 +48,21 @@ const handler = async (req: NextApiRequest) => {
       profileImageUrl = profileImage.previewURLs.medium;
     }
 
-    if (!profileImageUrl && profileToken?.definition?.media) {
-      const resultProfileImage = getPreviewUrls(profileToken.definition.media);
-      profileImageUrl = resultProfileImage?.large || resultProfileImage?.small || '';
+    const { media: profileMedia } = profileToken?.definition;
+
+    if (!profileImageUrl && profileMedia) {
+      profileImageUrl = getPreviewUrl(profileMedia);
     }
 
     const postToken = post.tokens?.[0];
     if (!postToken) {
-      return new ImageResponse(
-        (
-          <img
-            src={fallbackUrl}
-            style={{
-              width: 1200,
-              height: 630,
-              display: 'block',
-              objectFit: 'contain',
-            }}
-            alt="post"
-          />
-        ),
-        {
-          width: WIDTH_OPENGRAPH_IMAGE,
-          height: HEIGHT_OPENGRAPH_IMAGE,
-        }
-      );
+      return fallbackImageResponse;
     }
 
     let postImageUrl = '';
-    const resultPostImage = getPreviewUrls(postToken.definition.media);
-    if (!postImageUrl && resultPostImage?.large) {
-      postImageUrl = resultPostImage.large;
+    const { media: postMedia } = postToken?.definition;
+    if (!postImageUrl && postMedia) {
```

Comment:
hmm, `!postImageUrl` will always be true because `postImageUrl` is an empty string on L62

can just make this `if (postMedia)`